scripts/toolchain/dbtenv.sh

```diff
@@ -244,7 +238,7 @@ dbtenv_cleanup()
     
     # Kludgey but, quick path to getting rid of these
     printf "Removing any nuisance mac dotfiles..";
-    find toolchain | grep -e '\/[.][^\/]*$' | sed -e 's/\(.*\)/rm \"\1\"/' | /usr/bin/env bash;
+    find toolchain | grep -e '/[.][^\/]*$' | sed -e 's/\(.*\)/rm \"\1\"/' | /usr/bin/env bash;
```

Comment:
why is the escape removed?

Reply:
it was causing an error message on linux